modules/tinymce/src/plugins/template/test/ts/browser/DialogGetPreviewContentTest.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
```

Comment:
I'm guessing this was needed because of the long html strings below if you switch them to template strings and add new lines this can be removed.

Reply:
Couldn't really find a way to add newlines that didn't break the tests, unfortunately, so this will have to stay.

Reply:
At the very least you should just break the lines up using something like this:
```
const noCorsStyle = 'some really' +
  'long string';
```

Reply:
Worth noting to get the template idea to work, you'll probably need to delete the `indent: false` setting.